lib/active_record_shards/model.rb

```diff
@@ -9,13 +9,14 @@ def not_sharded
     end
 
     def is_sharded? # rubocop:disable Style/PredicateName
+      sharded_ivar = defined?(@sharded) ? @sharded : nil
```

Comment:
Won't it return `false` when the ivar is not defined in that case? This code cares about the difference between `nil` and `false`.